src/main/java/com/airlines/dao/DbConnector.java

```diff
@@ -0,0 +1,36 @@
+package com.airlines.dao;
+
+import com.zaxxer.hikari.HikariDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ResourceBundle;
+
+public class DbConnector {
+    HikariDataSource datasource;
+
+    public DbConnector() {
+        ResourceBundle resourceBundle = ResourceBundle.getBundle("db");
+        String url = resourceBundle.getString("url");
+        String username = resourceBundle.getString("username");
+        String password = resourceBundle.getString("password");
+
+        datasource = new HikariDataSource();
+        datasource.setJdbcUrl(url);
+        datasource.setUsername(username);
+        datasource.setPassword(password);
+    }
+
+    public Connection getDBConnection() {
+
+        Connection dbConnection = null;
+        try {
+            dbConnection = datasource.getConnection();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
```

Comment:
System.out.println is not used in commercial/production/real code.
Use log or throw pattern. We have agreed that logging is not using in order to reduce complexity and focus on dao layer, so you should throw your custom exception, check samples of code you did in jdbc practice, throw your own exception with "Can not obtain connection from datasource" or smth like that

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/3d7c7d3d96fe1b8bd3698ee76a32a9f3d5b5354e